src/docfx/build/context/Context.cs

```diff
@@ -180,15 +180,17 @@ internal sealed class Context : IDisposable
                 ContentValidator,
                 config,
                 errorLog);
+            var validatorExtension = new JsonSchemaValidatorExtension(
```

Comment:
if it's controlling content/metadata/build validations, how about just rename it to ValidationContext?

Reply:
rename to `CustomRuleProvider`